code/__HELPERS/_logging.dm

```diff
@@ -90,8 +90,7 @@
 //This replaces world.log so it displays both in DD and the file
 /proc/log_world(text)
```

Comment:
> //This replaces world.log so it displays both in DD and the file

might have been a leftover from another branch of mine, i'll take a look once im done with work today